app/components/display_rights_component.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class DisplayRightsComponent < ApplicationComponent
+  attr_reader :label,
+              :id
+
+  def initialize(id:)
+    @id = id
+    @label = WorkVersion::Licenses.label(id) if id.present?
```

Comment:
@srerickson will there even be a nil id? If so, we could use the `render?` method. This is a built-in feature of ApplicationComponent. Basically, if we were to initialize the component without an id, or a `nil` value, it won't display anything:

``` ruby
def render?
  id.present?
end
```

With the way it's written above, if the id were nil, the span html would still render and you'd get some kind of a broken link, I think.